### ISSUE TITLE
`text_` process support

### ISSUE THREAD

**Issue by soxofaan** [Open-EO/openeo-processes#329]
I just noticed this:

https://github.com/Open-EO/openeo-processes/blob/56b98ae3f2fd68cff3815c3885af3361e7c187fb/CHANGELOG.md#L15-L16

we actually added the `text_` processes this week in VITO backend: https://github.com/Open-EO/openeo-python-driver/issues/105

(sidenote: we implement `text_merge` instead of `text_concat` at the moment)

**Comment by m-mohr** [comment]
Uhm... so what's the action to take on side of the processes?

**Comment by soxofaan** [comment]
Well, it's not really necessary any more to move the `text_` processes to proposals.
Unless there are reasons to revise them.

To give a bit of background: we have an internal user that wants to do define a UDP with a `year` parameter and some processing along the lines of:
```python
year = Parameter.integer("year", description="Year")
previous_year = subtract(year, 1)
start_date = text_merge([previous_year, "-10-01"])
end_date = text_merge([year, "-12-31"])
cube = con.load_collection("TERRASCOPE_S2_TOC_V2", temporal_extent=[start_date, end_date])
```

**Comment by m-mohr** [comment]
Okay. As it's also available on GEE, we can indeed move it back.
Would it be ok for you to rename to `text_concat`? @soxofaan

**Comment by soxofaan** [comment]
> Would it be ok for you to rename to text_concat?

yes, I would also prefer `text_concat` or `text_join`  over `text_merge`
